install/prepare_site_init.md

```diff
@@ -142,6 +142,11 @@ with system-specific customizations.
 
     > **`IMPORTANT`** The CA may not be modified after install.
 
+#### Setup LDAP configuration
```

Comment:
Just add the "Setup LDAP configuration" and "End of LDAP configuration" to the ToC at the top, nested under the parent group.

Reply:
Added.